packages/fuel-gauge/src/policies.test.ts

```diff
@@ -157,8 +157,8 @@ describe('Policies', () => {
     } = launched;
 
     const factory = new ContractFactory(
-      ScriptMainArgsAbi__factory.bin,
-      ScriptMainArgsAbi__factory.abi,
+      PayableAnnotationAbiHex,
```

Comment:
Why were these changed to `PayableAnnotation`  abi?

Reply:
I assume because the `ScriptMainArgs` is not a contract and `PayableAnnotation` is.

Reply:
That's valid 😅 , perhaps it should be a script instance, although `ContractFactory` was used from this [test's inception](https://github.com/FuelLabs/fuels-ts/pull/1437/files#diff-0c6d881187bb1fdc9547309f7ee2830612689983980df36e3ec61be1224670b2R95), perhaps it was an oversight then as well. @Torres-ssf thoughts?

Reply:
@maschad I am using `PayableAnnotation` because the `ScriptMainArgs` ABI and bytes were being incorrectly used as a contract ABI and bytes.

Reply:
I suppose it doesn't really matter whether this is contract or a script for the purposes of this test, which is why it was passing before.

Reply:
@maschad The tests were passing because in both examples, the contract was never being deployed.

Reply:
This will become impossible once #2787 / #989 is done.